keps/sig-node/3570-cpumanager/README.md

```diff
@@ -156,11 +229,12 @@ type CPUSet map[int]struct{} // set operations and parsing/formatting helpers
 type CPUTopology // convenient type for querying and filtering CPUs
 ```
 
-#### Configuring the CPU Manager
+### Configuring the CPU Manager
 
-Kubernetes will ship with three CPU manager policies. Only one policy is
+Kubernetes will ship with CPU manager policies. Only one policy is
 active at a time on a given node, chosen by the operator via Kubelet
-configuration. The three policies are **none**, **static** and **dynamic**.
```

Comment:
Out of curisoity -- why was the `dynamic` policy removed?

Reply:
I don't have this information. I started contributing in this area way after the original discussion. I can *guess* the scope was downsized, and later the use case for this dynamic policy faded out.